R/render.r

```diff
@@ -162,6 +170,32 @@ check_open_graph <- function(og) {
   og[intersect(supported_fields, names(og))]
 }
 
+get_bs_version <- function(pkg = ".") {
+  pkg <- as_pkgdown(pkg)
+
+  template <- pkg$meta[["template"]]
+
+  if (is.null(template$bootstrap)) {
+    return(3)
+  }
+
+  if (!template$bootstrap %in% c(3, 4)) {
+    abort(
+      message = c(
+        "Boostrap version must be 3 or 4.",
+        i = sprintf("You used %s.", template$bootstrap),
+        paste0(
+          "Fix the `bootstrap` parameter in the configuration file, ",
```

Comment:
The hint is too long, I am not sure how to make it obvious where the parameter is (nested under template).
Maybe ultimately it should be a link to a vignette.

Reply:
#1488 will add a helper for such error messages cf https://github.com/r-lib/pkgdown/pull/1488#discussion_r572141468

Reply:
Let's finish off this PR next.

Reply:
rebased this PR to use `pkgdown_field()`